apps/console-lite/src/app/components/simple-market-list/simple-market-renderer.tsx

```diff
@@ -17,7 +17,9 @@ const MarketNameRenderer = ({ market, isMobile }: Props) => {
         )}
       />
       <div className="col-span-2 uppercase justify-start text-black dark:text-white text-ui-small md:text-market self-end">
-        {isMobile ? market.tradableInstrument.instrument.code : market.name}{' '}
+        {isMobile
```

Comment:
(non-blocking) wouldn't it be more elegant like `{market.tradableInstrument.instrument[isMobile ? 'code' : 'name']}`